### ISSUE TITLE
The World Oasis Starter Files Are Not The Same

### ISSUE THREAD

**Issue by MahdiRamezani8** [jonasschmedtmann/ultimate-react-course#91]
In the cabins feature folder has some new files and the other files are not the same.
![image](https://github.com/user-attachments/assets/9bd466e8-32af-490a-a983-2aaf6e3636aa)

**Comment by Emon-Khan** [comment]
True, I am facing problem while doing this project, in that specific project code is not synced properly as per the folder name......
